test/test_citusdb.yml

```diff
@@ -0,0 +1,256 @@
+---
+- hosts: citus
+
+  tasks:
+    - name: Install requirements for postgresql Ansible tasks
+      yum:
+        name: "{{ item }}"
+        state: present
+      with_items:
+        - python-psycopg2
+
+    - name: Download users data (31.7MB)
+#     shell: wget https://examples.citusdata.com/tutorial/users.csv -O /tmp/users.csv 
+      get_url:
+        url: https://examples.citusdata.com/tutorial/users.csv
+        dest: /tmp/users.csv
+        mode: 0755
+
+    - name: Download events data (74.7MB)
+#     shell: wget https://examples.citusdata.com/tutorial/events.csv -O /tmp/events.csv 
+      get_url:
+        url: https://examples.citusdata.com/tutorial/events.csv
+        dest: /tmp/events.csv
+        mode: 0755
+
+    - name: Ensure that a connection can be made
+      postgresql_exec:
+        login_user: "{{ pp_dba_username }}"
+        login_password: "{{ pp_dba_password }}"
+        login_host: citus_coordinator
```

Comment:
This could very based on context, so we should also set this to a variable, such as `login_host: "{{ citus_server_host }}"'.  This way we can use the same script to test both the Prod and Staging clusters, but not both if we choose.